pyte/screens.py

```diff
@@ -186,40 +186,27 @@ def __repr__(self):
     def display(self):
         """A :func:`list` of screen lines as unicode strings."""
         def render(line):
-            it = iter(line)
-            while True:
-                char = next(it).data
+            wide_char = False
```

Comment:
We cannot use iterators anymore, because of gaps in dict. And we can't skip next `for ... in` iteration without special state.